src/parser/jobs/pld/modules/Atonement.tsx

```diff
@@ -10,8 +10,8 @@ import React from 'react'
 
 // In seconds
 const USE_SEVERITY = {
-	95: TARGET.WARN,
-	100: TARGET.SUCCESS,
+	79: TARGET.WARN,
+	83: TARGET.SUCCESS,
```

Comment:
Any explanation for this change or?

Reply:
PLD can now drop 1 Atonement out of the 6 per minute that we generate in order to do the 61s rotation and keep buffs aligned. So 83% would now be the old 100%, and 79% the old 95%.